From: Markus Koschany <apo@debian.org>
Date: Tue, 13 Mar 2018 21:42:31 +0100
Subject: remove matherr

Work around "tclxmerr.c:40:30: error: dereferencing pointer to incomplete type
'struct exception'"

---
 src/tclx/src/tclxmerr.c | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/src/tclx/src/tclxmerr.c b/src/tclx/src/tclxmerr.c
index dc3596b..06aad3a 100644
--- a/src/tclx/src/tclxmerr.c
+++ b/src/tclx/src/tclxmerr.c
@@ -33,15 +33,6 @@
  *
  *-----------------------------------------------------------------------------
  */
-int
-matherr (except)
-    struct exception *except;
-{
-    if (Tcl_MathError (except->name, except->type))
-        return 1;
-    else
-        return 0;
-}
 
 #else
 
